lib/Hero/Hero.js

```diff
@@ -0,0 +1,122 @@
+import React, { lazy, Suspense } from 'react';
+import PropTypes, { element, elementType, node } from 'prop-types';
+import { Button as ButtonComponent } from '../Button';
+import Content from './Content';
+const Carousel = lazy(() =>
+  import('../vendor/EmblaCarousel/Carousel/Carousel'),
+);
+
+const Hero = ({
+  classNames = {},
+  title,
+  subTitle,
+  imageUrl,
+  hasOverlay,
+  NextImage,
+  slides,
+  Button = () => (
```

Comment:
Same comment as in `/lib/Hero/Content.js`. Let's not have a default `Button` component directly in here. That can be done in the story file.